apps/webapp/app/root.tsx

```diff
@@ -17,10 +17,12 @@ import { appEnvTitleTag } from "./utils";
 import { ErrorBoundary as HighlightErrorBoundary } from "@highlight-run/react";
 import { useHighlight } from "./hooks/useHighlight";
 import { ExternalScripts } from "remix-utils";
+import { withDevTools } from "remix-development-tools";
+import rdtStylesheet from "remix-development-tools/index.css";
 
 export const links: LinksFunction = () => {
-  return [{ rel: "stylesheet", href: tailwindStylesheetUrl }];
-};
+  return process.env.NODE_ENV === "development" ? [{ rel: "stylesheet", href: rdtStylesheet }, { rel: "stylesheet", href: tailwindStylesheetUrl }] : [{ rel: "stylesheet", href: tailwindStylesheetUrl }]
```

Comment:
Could we use a different env var for this? Let's call it `REMIX_DEVTOOLS_ENABLED` and the check should be `process.env.REMIX_DEVTOOLS_ENABLED === "1"`. Make sure to add that to the .env.example file at the root of the repo

Reply:
Sure, I'll change it!